documents/DeveloperGuide/MainPage.md

```diff
@@ -39,6 +39,16 @@ Additional options for the generation of MaterialX Python include the following:
 - `MATERIALX_PYTHON_OCIO_DIR`: Path to a folder containing the default OCIO configuration to be packaged with MaterialX Python. The recommended OpenColorIO configuration for MaterialX is [ACES 1.0.3](https://github.com/imageworks/OpenColorIO-Configs/tree/master/aces_1.0.3).
 - `MATERIALX_PYTHON_PYBIND11_DIR`: Path to a folder containing the PyBind11 source to be used in building MaterialX Python. Defaults to the included PyBind11 source.
 
+### Building MaterialX JavaScript
+
+Building the JavaScript and WebAssembly library requires the user to install and activate the emsdk environment. For more information follow the instructions in the JsMaterialX [README.md](../../source/JsMaterialX/README.md).
+
+The `MATERIALX_BUILD_JS` option will turn off the `MATERIALX_BUILD_RENDER` and the `MATERIALX_BUILD_TESTS` options.
```

Comment:
I was getting build errors when these libraries were included, this is why I disable them.

Reply:
Doing a WASM build needs to be separate due to Emscripten requirements -- It’s not possible to simultaneously (in a single `cmake configure`) do both a native build and a WASM build. The components for which no WASM support makes sense therefore need to be disabled during a WASM build.

Reply:
To make this more flexible, since the WASM build is now totally separated, can we just set 
`-DMATERIAL_BUILD_RENDER=OFF -DMATERIALX_BUILD_TEST=OFF` ? You might want to turn off any other modules not supported now since they take time to build but are never wrapped. e.g. All the ShaderGen modules. 

BTW, when I ran this I did find warnings showing up during the build. Could you log an issue for these and anything else that you find? We should fix them and then turn on warnings-as-errors.